spec/factories/countries.rb

```diff
@@ -22,16 +23,11 @@
     sequence :code, Country::CODES.cycle
 
     trait :requires_secondary_education_teaching_qualification do
-      sequence :code,
-               Country::CODES_REQUIRING_SECONDARY_EDUCATION_TEACHING_QUALIFICATION.cycle
+      subject_limited { true }
     end
 
     trait :doesnt_require_secondary_education_teaching_qualification do
-      sequence :code,
-               (
-                 Country::CODES -
-                   Country::CODES_REQUIRING_SECONDARY_EDUCATION_TEACHING_QUALIFICATION
-               ).cycle
+      subject_limited { true }
```

Comment:
Should this be `false`? Also I wonder if we even need this trait since the default value for this field is `false`.